migration/migrator/migrations/course/20191010120300_add_office_hours_queue.py

```diff
@@ -0,0 +1,25 @@
+"""Migration for the Submitty master database."""
+import json
+from pathlib import Path
+
+def up(config, database, semester, course):
+    database.execute("CREATE TABLE IF NOT EXISTS queue(entry_id serial PRIMARY KEY, user_id VARCHAR(20) NOT NULL, name VARCHAR (20) NOT NULL, time_in TIMESTAMP NOT NULL, time_helped TIMESTAMP, time_out TIMESTAMP, removed_by VARCHAR (20), status SMALLINT NOT NULL)");
+    database.execute("CREATE TABLE IF NOT EXISTS queue_settings(id serial PRIMARY KEY, open boolean NOT NULL, code VARCHAR (20) NOT NULL)");
+    database.execute("INSERT INTO queue_settings (open,code) VALUES (FALSE, '')");
```

Comment:
Is there every going to be more than one allowed queue open at a time?

Reply:
Yes. The plan is eventually to expand the queue to work in labs as well. Which often have both a help and checkpoint queue. As well as there are usually multiple independent lab sections being held at once that would want their own queues